packages/app-bridge/src/types/GuidelineAssetSettings.ts

```diff
@@ -0,0 +1,6 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Also here I would name the file `GuidelineAssetsSettings.ts` not `GuidelineAssetSettings.ts`